dashboard/test/factories/plc.rb

```diff
@@ -0,0 +1,56 @@
+FactoryGirl.allow_class_lookup = false
```

Comment:
In that case we should move those models into the pd namespace. The factory file name should match the model's namespace.